src/js/service/cloud-storage.js

```diff
@@ -66,76 +66,117 @@ silex.service.CloudStorage.prototype.ready = function(cbk) {
  * @param  {FileInfo} fileInfo
  * @param  {string} rawData
  * @param  {function()} cbk
- * @param  {?function(Object)=} opt_errCbk
+ * @param  {?function(Object, string)=} opt_errCbk
  */
 silex.service.CloudStorage.prototype.write = function(fileInfo, rawData, cbk, opt_errCbk) {
-  // save the data
-  this.ce.write(rawData, fileInfo)
-  .then(() => {
-    cbk();
-  })
-  .catch(e => {
-    console.error('Error: could not write file', fileInfo, e);
-    if (opt_errCbk) opt_errCbk(/** @type {Object} */ (e));
-  });
+  // // save the data
+  // this.ce.write(new Blob([rawData], {type: 'text/plain'}), fileInfo)
+  // .then(() => {
+  //   cbk();
+  // })
+  // .catch(e => {
+  //   console.error('Error: could not write file', fileInfo, e);
+  //   if (opt_errCbk) opt_errCbk(/** @type {Object} */ (e));
+  // });
+	const oReq = new XMLHttpRequest();
+	oReq.onload = function(event) {
+		if(oReq.status === 200) {
```

Comment:
What happened to your indentation? (╯°□°）╯︵ ┻━┻

Reply:
fixed